src/PinAssignment/PinAssignmentCreator.cpp

```diff
@@ -106,6 +107,11 @@ QString PinAssignmentCreator::targetDevice(
   return QString::fromStdString(projectManager->getTargetDevice());
 }
 
+QString PinAssignmentCreator::packagePinHeaderFile(ToolContext *context) const {
+  auto path = context->DataPath() / "etc" / "package_pin_info.json";
```

Comment:
Is this safe of different OSes ?

Reply:
Yes

Reply:
Does Qt handle the operator overloading across OSes ? Looking for documentation where that's described.

Reply:
it is not Qt. it is **std::filesystem::path**